### PR TITLE
elfutils 0.163

### PR DESCRIPTION
**Pull request by dholm**

### CODE REVIEW

**Comment by sjackman** [comment]
Do you have access to a Mac, and does `elfutils` compile on a Mac? If so, submit it upstream to https://github.com/Homebrew/homebrew

**Comment by dholm** [comment]
I do.  I'll see if it builds.

**Comment by dholm** [comment]
@sjackman `configure` fails with Clang due to missing support for the GNU extension [Nested Functions](https://gcc.gnu.org/onlinedocs/gcc/Nested-Functions.html):

    configure: error: gcc with GNU99 support required

**Comment by sjackman** [comment]
Add to the formula:
```ruby
fails_with :clang do
  build 700
  cause "gcc with GNU99 support required"
end
```

**Comment by dholm** [comment]
@sjackman Done.

**Comment by sjackman** [comment]
Does it build on Mac now?

File: Library/Formula/elfutils.rb [hunk 1]
```diff
@@ -0,0 +1,29 @@
+class Elfutils < Formula
+  desc "Libraries and utilities for handling ELF objects."
+  homepage "https://fedorahosted.org/elfutils/"
+  url "https://fedorahosted.org/releases/e/l/elfutils/0.163/elfutils-0.163.tar.bz2"
+  sha256 "7c774f1eef329309f3b05e730bdac50013155d437518a2ec0e24871d312f2e23"
+
+  depends_on "bzip2"
+  depends_on "xz"
+  depends_on "zlib"
```
Comment (sjackman) [inline Library/Formula/elfutils.rb]: Please change to
```ruby
  depends_on "xz"
  depends_on "bzip2" unless OS.mac?
  depends_on "zlib" unless OS.mac?
```

**Comment by sjackman** [comment]
The build fails with GCC 5.3.0:
```
checking for __thread support... no
configure: error: __thread support required
```

**Comment by sjackman** [comment]
Merged. Thanks for your contribution, David. Sorry for the extended delay.

**Comment by sjackman** [comment]
Just out of curiosity, what do you use `elfutils` for instead of `binutils`?

**Comment by dholm** [comment]
@sjackman elfutils provides libraries that are used by many other applications, such as [perf](https://perf.wiki.kernel.org/index.php/Main_Page).

**Pull request closed without merging**